py/torch_tensorrt/dynamo/conversion/impl/activation/ops.py

```diff
@@ -0,0 +1,402 @@
+import math
+from typing import Any, Optional, Tuple
+
+import numpy as np
+import tensorrt as trt
+import torch
+from torch import Tensor
+from torch.fx.node import Target
+from torch_tensorrt.dynamo._SourceIR import SourceIR
+from torch_tensorrt.fx.converters.converter_utils import (
+    get_trt_plugin,
+    mark_as_int8_layer,
+    set_layer_name,
+)
+from torch_tensorrt.fx.types import TRTNetwork, TRTPluginFieldCollection, TRTTensor
+
+from .base import convert_activation
+
+
+def gelu(
+    network: TRTNetwork,
+    target: Target,
+    source_ir: Optional[SourceIR],
+    name: str,
+    input_val: TRTTensor,
+    alpha: Optional[Any] = None,
+) -> TRTTensor:
+    approximate = alpha
+    if approximate is not None:
+        raise RuntimeError("GeLU converter currently doesn't support fast gelu compute")
+    if not isinstance(input_val, TRTTensor):
+        raise RuntimeError(
+            f"GELU received input {input_val} that is not part "
+            "of the TensorRT region!"
+        )
+    if network.has_implicit_batch_dimension:
+        raise RuntimeError(
+            "GeLU converter currently doesn't support implicit batch dimension"
+        )
+    plugin_name = "CustomGeluPluginDynamic"
+    # type_id 0 for float32, 1 for  float16
+    type_id = trt.PluginField(
+        "type_id", np.array(0, dtype=np.int32), trt.PluginFieldType.INT32
+    )
+    field_collection = TRTPluginFieldCollection([type_id])
+    plugin_version = "1"
+
+    plugin = get_trt_plugin(plugin_name, field_collection, plugin_version)
+
+    layer = network.add_plugin_v2([input_val], plugin)
```

Comment:
Should use a lowering pass and run it natively instead of in a plugin

Reply:
There is currently an `aten.gelu` plugin enabled on `main`, so this can potentially be removed.